index.js

```diff
@@ -55,17 +55,26 @@ class NpmAddDependencies {
   };
 
   runNpmShow(dep) {
+    const [depName, depVersion] = dep.split('@');
     return new Promise((resolve) => {
       npmRun.exec(`npm show ${dep} dist-tags`, (err, stdout) => {
         if (!err) {
           const parsed = stdout.match(/latest: '(.*?)'/i);
 
           if (!parsed || undefined === parsed[1]) {
-            console.error(`Could not obtain the latest version for: ${dep}. Skip.`);
+            if (undefined === depVersion) {
+              console.error(`Could not obtain the latest version for: ${dep}. Skip.`);
```

Comment:
Here can be one `console.log` with the ternary operator inside instead of if...else..., what do you think?

```js
console.error(`Could not obtain the ${depVersion ? 'specified' : 'latest'} version for: ${depName}. Skip.`);
```